packages/components/nodes/chatmodels/AWSBedrock/AWSChatBedrock.ts

```diff
@@ -39,6 +41,12 @@ class AWSChatBedrock_ChatModels implements INode {
             optional: true
         }
         this.inputs = [
+            {
+                label: 'Cache',
+                name: 'cache',
+                type: 'LLMCache',
```

Comment:
Can we find and replace all `LLMCache` with `BaseCache`

Reply:
Sorry, that was sloppy on my end. :-) I have changed all and pushed the new commit.